src/ImpactX.H

```diff
@@ -27,8 +28,14 @@ namespace impactx
         : public amrex::AmrCore
     {
       public:
-        ImpactX () = default;
+        ImpactX (amrex::Geometry const& geom, amrex::AmrInfo const& amr_info);
         ~ImpactX () = default;
+        ImpactX (ImpactX const&) = delete;
+        ImpactX (ImpactX &&) = delete;
+        void operator= (ImpactX const&) = delete;
+        void operator= (ImpactX &&) = delete;
+
+        void initData ();
```

Comment:
Can't let the perfect Doxygen streak slip:
```suggestion
        /** Initialize particle containers and MultiFabs (fields) */
        void initData ();

        /** Run the main simulation loop */
        void evolve ();
```